ignite/config/plugins/parse.go

```diff
@@ -53,13 +53,13 @@ func ParseDir(dir string) (*Config, error) {
 }
 
 var (
-	// filenames is a list of recognized names as Ignite's plugins config file.
-	filenames       = []string{"plugins.yml", "plugins.yaml"}
-	defaultFilename = filenames[0]
+	Filenames = []string{"igniteapps.yml", "igniteapps.yaml"}
```

Comment:
any reason this is exported?

Reply:
Nice catch! No reason, there is no need for it to be changed to public anymore